classes/player/Player.py

```diff
@@ -84,7 +91,7 @@ def send_join_packets(self, player: Player):
         # 'brand'
         NetworkController.send_packet_player(player.entity_id, PacketType.PLUGIN_MESSAGE, {
             'channel': 'minecraft:brand',
-            'data': 'McPy/0.0.1-alpha',
+            'data': 'McPy/' + version,
```

Comment:
We could just as well calculate the brand data beforehand, that is before the method is run for the extra speed.

Reply:
So should we leave this the way it was for speed? And then just change it on a new update?

Reply:
the version tag is already calculated beforehand, so we can bundle these two actions

Reply:
It would probably just be easier if we followed what everyone else does and just make it `McPy`.